src/profile-management/src/employees/employees.service.ts

```diff
@@ -1,44 +1,44 @@
 import { Injectable } from '@nestjs/common';
-import { EmployeesRepository } from './employees.repository';
-import { Employee } from '../shared/schemas/employee.schema';
+import { CreateEmployeeDto } from './dto/create-employee.dto';
 import { UpdateEmployeeDto } from './dto/update-employee.dto';
+import { EmployeesRepository } from './employees.repository';
+import { Employee } from './schemas/employee.schema';
 import { FilterQuery } from 'mongoose';
-import { hashPassword } from 'src/shared/helper';
-import { CreateEmployeeDto } from './dto/create-employee.dto';
+import { hashPassword } from 'src/employees/helper';
 
 @Injectable()
 export class EmployeesService {
   constructor(private readonly employeesRepository: EmployeesRepository) {}
 
-  async getEmployeeById(id: string): Promise<Employee> {
-    return this.employeesRepository.findOne({ _id: id });
-  }
-
-  async getEmployeeByPersonalId(personalId: string): Promise<Employee> {
-    return this.employeesRepository.findOne({ personalId: personalId });
+  async createEmployee(
+    createEmployeeDto: CreateEmployeeDto,
+  ): Promise<Employee> {
+    createEmployeeDto.hash = await hashPassword(createEmployeeDto.hash);
+    return this.employeesRepository.create({ createEmployeeDto });
   }
 
   async getEmployees(): Promise<Employee[]> {
     return this.employeesRepository.find({});
```

Comment:
try adding this `{}` as default value for the parameter so you don't have to pass around empty objects